src/praisonai/praisonai/bots/_session.py

```
@@ -798,3 +798,48 @@ def active_sessions(self) -> int:
+        config: BotConfig instance with session configuration
+        platform: Platform identifier (e.g., "telegram", "slack")
+        run_control: Optional run control for Telegram (keyword-only)
+    
+    Returns:
+        Configured BotSessionManager instance
+    """
+    # Try to get the default session store
+    try:
+        from praisonaiagents.session import get_default_session_store
+        store = get_default_session_store()
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)